src/variables/components/TypeAheadVariableDropdown.test.tsx

```diff
@@ -71,7 +71,7 @@ const setInitialState = (state: AppState): AppState => {
             values: {
               '03cbdc8a53a63000': {
                 values,
-                selected: ['defbuck'],
+                selected: [''],
```

Comment:
The new behavior is to show only the selected item in the list, the whole list is shown after the input is cleared. I think this should be changed in the future as that isn't how dropdown's work.

Reply:
I agree with this take and I think that would be a positive UX addition. Adding to that, we might consider changing the behavior shown at 0:17 in the video - rather than closing the dropdown when removing the selected item, it might be better UX to keep it open and show the newly non-filtered list. This isn't something to block this PR on, mind. Just a suggestion for future work.

Reply:
I agree! I can add that in a future clockface PR 👍

Reply:
These suggestions have been added to the Clockface!